woocommerce/woocommerce-gutenberg-products-block
# Fix "Product Price" block margin on the "All products" block

Fixes the margin of the `Product price` block, more context: https://github.com/woocommerce/woocommerce-blocks/issues/8511

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/8511

### Testing
1. Set the `woocommerce_blocks_phase = 3` in the `blocks.ini`.
2. Create a new test page and add the `All Products` block.
3. Click on the pencil icon to edit the layout, click the `Product price` block and change the block margin.
4. Verify the margin is applied in the editor and click `Done`.
5. Add the `Products` block, click on the `Product price`, and change the block margin.
6. Verify the margin is applied in the editor.
7. Save the page, go to the frontend, and verify the margin is also applied there for both blocks.
8. Set the `woocommerce_blocks_phase = 1` in the `blocks.ini` file, repeat the steps from 1-7.

### WooCommerce Visibility



### Changelog

> Fix: make `Price product` margin work in the `All products` block.
